src/http/headers.cr

```diff
@@ -247,11 +247,11 @@ struct HTTP::Headers
     end
   end
 
-  def get(key) : Array(String)
+  def get(key : HTTP::Headers::Key | String) : Array(String)
```

Comment:
ditto

Reply:
I think I missed the original comment to this one :sweat_smile:

Reply:
This reference comes from [here](https://github.com/crystal-lang/crystal/blob/master/src/http/headers.cr#L305) (`pretty_print` method below that's iterating through the keys of `@hash`).,